app/helpers/spaces_helper.rb

```diff
@@ -6,14 +6,30 @@ def edit_field(partial, form)
   end
 
   def inline_editable(field, title_tag = :h2, title_text = Space.human_attribute_name(field), &block)
-    render partial: 'spaces/edit/editable_inline', locals: {
+    render partial: 'spaces/edit/common/editable_inline', locals: {
       field: field,
       title_tag: title_tag,
       title_text: title_text,
       block: block
     }
   end
 
+  def render_space_and_owner_field(space, field)
+    # rubocop:disable Style/IfUnlessModifier
+    field_in_space = (if space.respond_to?(field) && space.public_send(field).present?
+                        space.public_send(field)
+                      end)
+    field_in_space_owner = (if space.space_owner.respond_to?(field) && space.space_owner.public_send(field).present?
+                              space.space_owner.public_send(field)
+                            end)
```

Comment:
This looks wierd to me, should it probably be like this?
```rb
field_in_space = space.public_send(field) if space.respond_to?(field) && space.public_send(field).present?
field_in_space_owner = space.space_owner.public_send(field) if space.space_owner.respond_to?(field) && space.space_owner.public_send(field).present?
```

Reply:
Rubocop didn't like the long lines, so I tried to break them up, but agreed that it looks weirder this way... I could, I guess, break it up like this:

```ruby
    owner = space.space_owner
    # Setting the fields only if they are available (respond_to) and they have any content (present?)
    field_in_space = (space.public_send(field) if space.respond_to?(field) && space.public_send(field).present?)
    field_in_space_owner = (owner.public_send(field) if owner.respond_to?(field) && owner.public_send(field).present?)

```

The paranthesis is needed to return nil as field_in_space, else ruby interprets it as "(field_in_space = foo) if bar", and  we end up with field_in_space possibly being undefined.

Reply:
Tried fixing it like that, what do you think?